### DISCUSSION TITLE
Can subfolders please show in the main Assets pane?

### DISCUSSION THREAD

**Discussion by adrienne** [craftcms/cms#10216] [Ideas]
We've got nontechnical clients who are extremely confused, all the time, about having to tree down in the sidebar to get to subfolders. I had to send this screenshot earlier today:
![image](https://user-images.githubusercontent.com/89705/144898053-230e149e-b4fe-4d55-904d-a469d9cc4894.png)
 
It would be nice if the main pane looked (at least optionally) like a "traditional" file explorer, with subfolders.

**Comment by mmikkel** [comment]
+100, there's a similar feature request [here](https://github.com/craftcms/cms/discussions/9171)

This particular detail in Craft has always been a struggle for authors. Ideally I agree the Assets index should look and function more like a traditional file explorer (or, perhaps that a traditional file explorer view could be added in addition to the current thumbnails and table views) – but even a quick fix like adding actual folder icons to the sidebar (instead of/next to the arrows), would probably help a lot.
